daprdocs/content/en/operations/components/setup-bindings/supported-bindings/rabbitmq.md

```diff
@@ -40,6 +42,7 @@ spec:
 - `ttlInSeconds` is an optional parameter to set the [default message time to live at RabbitMQ queue level](https://www.rabbitmq.com/ttl.html). If this parameter is omitted, messages won't expire, continuing to exist on the queue until processed.
 - `prefetchCount` is an optional parameter to set the [Channel Prefetch Setting (QoS)](https://www.rabbitmq.com/confirms.html#channel-qos-prefetch). If this parameter is omiited, QoS would set value to 0 as no limit.
 - `exclusive` determines whether the topic will be an exclusive topic or not
+- `maxPriority` is an optional parameter to set the [priority queue](https://www.rabbitmq.com/priority.html). If this parameter is omitted, queue will be created as a general queue instead of a priority queue.
```

Comment:
If would be useful to give an example here. For example what does setting this to "5" mean? Is this the priority level of the message, which when set with the priority parameter is treated as a high priority message?

Reply:
Yes, it's sets the maximum priority on the queue. it's further explained in the URL linked here.